<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeControls</class>
 <widget class="QDockWidget" name="DateTimeControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>273</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>308</width>
    <height>273</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>524287</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Date Time Controls</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="date_label">
      <property name="text">
       <string>Date (yyyy-mm-dd)</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="date_edit">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="time_label">
      <property name="text">
       <string>UTC Time (hh:mm:ss)</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="time_edit">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="setDateButton">
      <property name="text">
       <string>Set Date</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="Line" name="line1_4">
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QCheckBox" name="enableTimeUpdates_chkbx">
      <property name="text">
       <string>Enable Real Time Ephemeris Updates</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="Line" name="line1_3">
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="mpsLabel">
      <property name="text">
       <string>Fast Time (Min Per Sec)</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="fastTimeLineEdit">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QPushButton" name="updateInFastTimeButton">
      <property name="text">
       <string>Start Fast Time</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>setDateButton</sender>
   <signal>clicked()</signal>
   <receiver>DateTimeControls</receiver>
   <slot>setDate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateInFastTimeButton</sender>
   <signal>clicked()</signal>
   <receiver>DateTimeControls</receiver>
   <slot>fastTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setDate()</slot>
  <slot>fastTime()</slot>
 </slots>
</ui>
